packages/eas-build-job/src/logs.ts

```diff
@@ -4,6 +4,7 @@ export enum BuildPhase {
   SPIN_UP_BUILDER = 'SPIN_UP_BUILDER',
   BUILDER_INFO = 'BUILDER_INFO',
   READ_APP_CONFIG = 'READ_APP_CONFIG',
+  READ_PACKAGE_JSON = 'READ_PACKAGE_JSON',
```

Comment:
make sure to add that on the website first otherwise it will be displayed as READ_PACKAGE_JSON or unknown phase (I don't remember exactly)

Reply:
Yes, did done that, here's the companion PR: https://github.com/expo/universe/pull/9296

Reply:
Yup, see mentioned PR below.